README.rst

```diff
@@ -47,6 +49,14 @@ Getting Started
 
 .. _Running server on device: #browsing-server-on-device
 
+Issues
+---------------
+For questions on development for the App you can post a topic here 
+https://groups.google.com/forum/#!forum/zulip-android
+
+For questions on development for the Server you can post a topic here 
+https://groups.google.com/forum/#!forum/zulip-devel
```

Comment:
yeah then let's do it!  I think it's best to optimize for making it easy for new contributors to get started.